main.go

```diff
@@ -31,6 +31,7 @@ func main() {
 
 func Main() int {
 	config.Init()
+	log.Init()
```

Comment:
Why did you move it here ?

Reply:
mainly to be explicit and avoid magic `init` functions

Reply:
ok